CharacterCreation/Class-Specific Documentation/Doctor Procedures.txt

```diff
@@ -325,9 +327,10 @@ Range:		Payload (use in combination with other powers)
     A biologic target afflicted with this poison must succeed on a DC 110 FORT 
 save or the target takes 180 damage that ignores shields and armor and silences 
 the target. If the target succeeds on their save, they take 60 damage instead. A 
-silenced target is prevented from using nanite abilities for 10 turns. (Also 
-prevents Engineering Processes and Doctor Procedures due to violent tremors). At 
-level 13, damage increases to 220.
+silenced target is prevented from using abilities that use Nanites for 10 turns. 
```

Comment:
We didn't add silenced to the Glossary yet, did we?
Regardless, I think "silences the target" should be changed because the target isn't speaking spells (anymore). Can you rephrase the text to remove or rephrase  'silences'/'silenced'?

Reply:
"Silence" does sound out of aesthetic, but we should definitely have a word for the state of not being able to use our nanite abilities. How about Suppressed?

Reply:
"Suppressed" is already taken for being under the effects of Light/Med/Heavy Suppression by a typical ranged weapon. Although I think Suppressed would be a great fit here, as well.

What about "Disrupted"? (Does that have overlap with hacking/robot disabling of some kind?)

Or maybe some term that signifies the inability for the target to calm their arms/physical nerves...?

Reply:
P.S. I'm not sure if "Suppressed" is added to the glossary yet

Reply:
Totally forgot about suppression levels :(
How about Stifled?

Reply:
This is the only part of the game (as far as I know) where the Silence effect is used. I don't know that it makes sense to encapsulate it as a status effect? Would other classes/enemies/hazards use a similar Condition?  

If we can make it more aesthetically intuitive by renaming it, we should do that. The reason for the original name of "silenced" comes from the standard industry term for such abilities as used in WoW, LoL, CoH, etc where the term is used to mean "cannot cast spells". Tactically, I think such an ability makes a lot of sense for the medic to have, and I would like to update/polish the ability until it makes sense and works well rather than yanking it. 
Looking forward to more feedback. If I'm understanding this right, this is the last blocker to updating the Doctor class?